File: app/src/main/java/com/patloew/template/ui/main/MainMvvm.java
package com.patloew.template.ui.main;

import com.patloew.template.ui.base.view.MvvmView;
import com.patloew.template.ui.base.viewmodel.MvvmViewModel;

public interface MainMvvm {

    interface View extends MvvmView {

    }

    interface ViewModel extends MvvmViewModel<View> {

    }
}

File: app/src/main/java/com/patloew/template/ui/main/MainMvvm.kt
package com.patloew.template.ui.main

import com.patloew.template.ui.base.view.MvvmView
import com.patloew.template.ui.base.viewmodel.MvvmViewModel

interface MainMvvm {

    interface View : MvvmView

    interface ViewModel : MvvmViewModel<View>
}
